analysis/report_templates/default.html

```diff
@@ -228,21 +233,47 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                             </div>
                         </div> <!-- row -->
 
+                        {% if detailed_coverage_report%}
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by rare region covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.rare_region_ranking_plot }}">
+                                    The table shows the coverage rank of each fuzzer
+                                    on the rare regions for this benchmark. The rare
+                                    region is defined as only being hit by less than
+                                    threshold(default=20%) of all fuzzers.
```

Comment:
Is the default in fact 20% I think it's 0%.
To be honest, I'm not sure I'm a fan of using the term "rare". While "rare" is generic so it can be used to mean different thresholds, I think it would be far clearer if we said unique and assumed a threshold of 0. Do others agree that we should drop functionality to make things more understandable?

Reply:
+1